inc/Engine/CDN/RocketCDN/AdminPageSubscriber.php

```diff
@@ -110,6 +114,10 @@ public function display_rocketcdn_status() {
 	 * @return array
 	 */
 	public function rocketcdn_field( $fields ) {
+		if ( rocket_get_constant( 'WP_ROCKET_WHITE_LABEL_ACCOUNT' ) ) {
```

Comment:
@Tabrisrp Let's move this code into a private method to encapsulate the guard clause.

```php
if ( $this->is_while_label_account() ) {
```

and 

```php
private function is_while_label_account() {
	return (bool) rocket_get_constant( 'WP_ROCKET_WHITE_LABEL_ACCOUNT', false );
}
```

and then update all of the uses in this class.

Why?

- Readability
- Encapsulate the knowledge
- If ever we need to change it, we can do so in one method.